data/projects/a/arbitrum_dao.yaml

```diff
@@ -0,0 +1,16 @@
+version: 7
+name: Arbitrum DAO in Europe
```

Comment:
Should follow naming conventions ... this should have a `filename` of 'public-goods-portal.yaml' and a `name` of 'public-goods-portal'